src/components/DpSelect/DpSelect.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta } from '@storybook/addon-docs'
+import DpSelect from './DpSelect'
+
+<Meta
+  title="Components/Select"
+  component={DpSelect} />
+
+# DpSelect
+
+The DpSelect component provides `<select>` element to create a drop-down list.
+
+With `label` attribute can select menu title be passed, with `options` attribute can call a computed.
+
+And with `@select` event can call a method.
+
+## Default usage
+
+```html
+<dp-select
+  v-model="selectModel"
+  class="u-mb-0_5"
```

Comment:
the class does not add value in the example.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/fcd7516cb6e2df5ee35e745788547fcfc2a1d57c